scripts/time_plots.py

```diff
@@ -0,0 +1,80 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import sys
+import json
+
+# Load data
+filename = sys.argv[1]
+
+# Make DataFrame to facillitate plotting
+if filename.endswith(".json"):
+    with open(filename) as json_file:
+        data = json.load(json_file)
+    unit = data["units"]
+
+    # Get Unit
+    if unit == "Meters":
+        abbreviation = "m"
+    elif unit == "Feet":
+        abbreviation = "ft"
+    elif unit == "Inches":
+        abbreviation = "in"
+    elif unit == "Degrees":
+        abbreviation = "deg"
+    elif unit == "Rotations":
+        abbreviation = "rot"
+    else:
+        raise ValueError("Invalid Unit")
+
+    # Make Columns
+    columns = ["Timestamp (s)", "Test"]
+    if "Drive" in data["test"]:
+        columns.extend([
+            "Left Volts (V)",
+            "Right Volts (V)",
+            f"Left Position ({abbreviation})",
+            f"Right Position ({abbreviation})",
+            f"Left Velocity ({abbreviation}/s)",
+            f"Right Velocity ({abbreviation}/s)",
+            "Gyro Position (deg)",
+            "Gyro Rate (deg/s)",
+        ])
+        unit_columns = columns[4:8]
+    else:
+        columns.extend([
+            "Volts (V)", f"Position ({abbreviation})",
+            f"Velocity ({abbreviation}/s)"
+        ])
+        unit_columns = columns[3:]
+
+    df = pd.DataFrame(columns=columns)
+    for test in (test for test in data.keys() if "-" in test):
+        df_data = [[pt[0]] + [test] + pt[1:] for pt in data[test]]
+        df = df.append(pd.DataFrame(df_data, columns=columns))
```

Comment:
Isn't assigning to `df` redundant here?
```
>>> df=["a"]
>>> df.append("b")
>>> df
['a', 'b']
```

Reply:
by default the append method returns a copy, but I can set the in_place param to True if that would reduce the redundancy